### PR TITLE
Add mkdocs-marimo

### PR DESCRIPTION
**Pull request by pavelzw**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pavelzw** [comment]
@conda-forge/help-python this is ready for review

File: recipes/mkdocs-marimo/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+- url: https://pypi.io/packages/source/${{ name[0] }}/${{ name }}/mkdocs_marimo-${{ version }}.tar.gz
+  sha256: 9ac2fb08850f2f0f5ab01d06673b18425e678be660d5493ab7e7810993e60fce
+
+build:
+  script: python -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/mkdocs-marimo/recipe.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```

Comment (pavelzw) [reply recipes/mkdocs-marimo/recipe.yaml:14]: i thought in v1 recipes we don't use `{{ PYTHON }}` anymore? https://github.com/conda-forge/staged-recipes/blob/a8e5dbc12b38de1a6472d983399ac88c1ebfcd99/recipes/example-v1/recipe.yaml#L50

Comment (ocefpaf) [reply recipes/mkdocs-marimo/recipe.yaml:14]: Not sure. We still don't have docs on that for reviewers. I believe the extra options there are still welcomed though.

**Pull request merged**